FileAPI/snapshot-state/content-size-manual.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<label>Select a file first: <input type="file" id="fileInput"></label><br>
+<button id="readyButton">Then modify the content of the selected file and click this</button><br>
```

Comment:
Be explicit about the required modification (i.e. the size should be smaller)

Reply:
No, the size can be random. Larger size is also okay.